app/table/cells/DefaultCell.tsx

```diff
@@ -5,14 +5,8 @@
  *
  * Copyright Oxide Computer Company
  */
-import type { CellContext } from '@tanstack/react-table'
-
 import type { Cell } from './Cell'
 
 export const DefaultCell = ({ value }: Cell<string>) => (
   <span className="text-secondary">{value}</span>
 )
-
-export const defaultCell = <T, U extends React.ReactNode>(props: CellContext<T, U>) => (
-  <span className="text-secondary">{props.getValue()}</span>
-)
```

Comment:
Turns out we don't need this. Worth keeping an eye out for situations where it behaves differently because it's not in a span, but that seems pretty unlikely.

https://github.com/oxidecomputer/console/blob/9485ca2392ba7e4e5b65d370e4020a95ad46b1bd/app/ui/styles/components/table.css#L31-L36